routers/private/serv.go

```diff
@@ -291,3 +344,27 @@ func ServCommand(ctx *macaron.Context) {
 	ctx.JSON(http.StatusOK, results)
 	// We will update the keys in a different call.
 }
+
+func pushCreateRepo(authUser, owner *models.User, repoName string) (*models.Repository, error) {
```

Comment:
Since this function is identical in serv.go and http.go, can it be moved to maybe services instead?